e2e/specs/startup/welcome_screen_modal.test.js

```diff
@@ -0,0 +1,142 @@
+
+    it('MM-T4975_4 should be able to move forward through slides automatically every 5 seconds', async () => {
+        const welcomeSlideClass = await welcomeScreenModal.getAttribute('#welcome', 'class');
+        welcomeSlideClass.should.contain('Carousel__slide-current');
+
+        await asyncSleep(5500);
+
+        const channelSlideClass = await welcomeScreenModal.getAttribute('#channels', 'class');
+        channelSlideClass.should.contain('Carousel__slide-current');
+    });
+
+    it('MM-T4975_5 should show the slides in the expected order', async () => {
+        const welcomeSlideClass = await welcomeScreenModal.getAttribute('#welcome', 'class');
+        welcomeSlideClass.should.contain('Carousel__slide-current');
+
+        await welcomeScreenModal.click('#nextCarouselButton');
+
+        const channelSlideClass = await welcomeScreenModal.getAttribute('#channels', 'class');
+        channelSlideClass.should.contain('Carousel__slide-current');
+
+        await welcomeScreenModal.click('#nextCarouselButton');
+
+        const playbooksSlideClass = await welcomeScreenModal.getAttribute('#playbooks', 'class');
+        playbooksSlideClass.should.contain('Carousel__slide-current');
+
+        await welcomeScreenModal.click('#nextCarouselButton');
+
+        const boardsSlideClass = await welcomeScreenModal.getAttribute('#boards', 'class');
+        boardsSlideClass.should.contain('Carousel__slide-current');
+    });
+
+    it('MM-T4975_6 should be able to move from last to first slide', async () => {
+        await welcomeScreenModal.click('#PaginationIndicator3');
+
+        const boardsSlideClass = await welcomeScreenModal.getAttribute('#boards', 'class');
+        boardsSlideClass.should.contain('Carousel__slide-current');
+
+        await welcomeScreenModal.click('#nextCarouselButton');
+
+        const welcomeSlideClass = await welcomeScreenModal.getAttribute('#welcome', 'class');
+        welcomeSlideClass.should.contain('Carousel__slide-current');
+    });
+
+    it('MM-T4975_7 should be able to move from first to last slide', async () => {
+        const welcomeSlideClass = await welcomeScreenModal.getAttribute('#welcome', 'class');
+        welcomeSlideClass.should.contain('Carousel__slide-current');
+
+        await welcomeScreenModal.click('#prevCarouselButton');
+
+        const boardsSlideClass = await welcomeScreenModal.getAttribute('#boards', 'class');
+        boardsSlideClass.should.contain('Carousel__slide-current');
+    });
+
+    it('MM-T4975_8 should be able to click the get started button and be redirected to new server modal', async () => {
+        await welcomeScreenModal.click('#getStartedWelcomeScreen');
+
+        const newServerModal = await this.app.waitForEvent('window', {
+            predicate: (window) => window.url().includes('newServer'),
+        });
+        const modalTitle = await newServerModal.innerText('#newServerModal .modal-title');
+        modalTitle.should.equal('Add Server');
+    });
```

Comment:
Okay, I've see Steve added bunch of test cases in Zephyr, just need to match the test case IDs.
<img width="1059" alt="Screen Shot 2022-08-12 at 7 23 46 PM" src="https://user-images.githubusercontent.com/5334504/184344615-dabe8351-ad24-429a-bda3-edc6e124ba11.png">